input/pages/09-b-cqlreference.md

```diff
@@ -6137,6 +6140,8 @@ define "ValuesetContainsNullString": "Acute Pharyngitis" ~contains null as Strin
 > Note carefully that the use of the string overloads for membership testing in code systems and value sets is potentially problematic and should be used with caution, if at all
 {: .note-warning}
 
+{: .in_valueset}
```

Comment:
There is already an anchor here from the header, no need for this change:
https://cql.hl7.org/2025Sep/09-b-cqlreference.html#in-valueset